File: April-18/23-04-18/python/01-Little-Jhool-and-his-punishment.py
for _ in range(int(input())):
    n = int(input())
    b, g = map(int,input().split())
    
    if(max(b,g) - 1 > min(b,g)):
        print("Little Jhool wins!")
    else:
        print("The teacher wins!")
    

File: April-18/23-04-18/java/01-Little-Jhool-and-his-punishment.java

import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int t = input.nextInt();        
        while(t-- > 0)
        {
            int n, b, g;
            n = input.nextInt();
            b = input.nextInt();
            g = input.nextInt();
            
            if(Math.max(b, g) - 1 > Math.min(b,g))
                System.out.println("Little Jhool wins!");
            else
                System.out.println("The teacher wins!");
        }
    }
}
